src/OgContextInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Drupal\og;
+
+/**
+ * @file
+ * Contains \Drupal\og\OgContextInterface.
+ */
+
+use Drupal\Component\Plugin\PluginInspectionInterface;
+use Drupal\Core\Plugin\Context\ContextProviderInterface;
+
+/**
+ * Defines an interface for OG context plugins.
+ */
+interface OgContextInterface extends PluginInspectionInterface, ContextProviderInterface {
+
+  /**
+   * Return the current group. If no group will be found return Null.
+   *
+   * @return ContentEntityBase
+   *   Return the best matching group.
+   */
+  public function getGroup();
```

Comment:
This name should probably be `OgGroupResolverInterface` as was suggested earlier. The plugins try to resolve a group from the current request state, they don't actually provide a context by themselves.